view/frontend/templates/authorization.phtml

```diff
@@ -62,32 +59,42 @@ $checkoutKey = $block->getCheckoutKey();
             scriptTag.setAttribute('data-publishable-key', '<?= /* @noEscape */ $checkoutKey; ?>');
             document.head.appendChild(scriptTag);
         }
+
+        let injectButtonsWhenScriptLoaded = function() {
+            if (typeof BoltAccount === 'undefined') {
+                window.setTimeout(insertButton, 100);
```

Comment:
Can we avoid setTimeout?
Please check this approach https://github.com/BoltApp/bolt-magento2/blob/master/view/frontend/templates/js/boltglobaljs.phtml#L59